README.md

```diff
@@ -1 +1,128 @@
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [TO-DO-LIST] <a name="about-project"></a>
+
+My to-do list project in Microverse.
+
+**[TO-DO-LIST]** is a project experience that Microverse offers to help nuture your skills in designing on different screen sizes.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 1. HTML5
+ 2. CSS3
+ 3. JavaScript
+
+
+### Key Features <a name="key-features"></a>
+
+  1. It is made to cater for mobile devices and desktops
+  2. Introductory page
+  3. Preview of the list page
+  4. Responsive design
+
+### Live Demo <a name="live-demo"></a>
+
+[Live Demo](not available)
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites
+
+In order to run this project you need:
+
+An extension to your browser
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone [TO-DO-LIST](https://github.com/2004-okari/TODOLIST)
+### Install
+
+Install this project with:
+
+cd MY-PORTFOLIO
+npm install
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file on the browser or use a VS code extension server to start the project`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment
+
+Will deploy the project using github pages.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+👤 **2004-okari**
+
+- GitHub: [2004-Okari](https://github.com/2004-okari)
+- LinkedIn: [Okari-Nyandika](https://www.linkedin.com/in/rooney-okari-86a5ba250/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+1: Details modal 
+2. Deployment
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/2004-okari/TODOLIST/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> 
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project you can star the repository to make it look better.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me all the necessary knowledge i need to make this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] When this link is clicked it shows a 404 error page, which is not the appropriate license file that should be displayed please kindly provide the appropriate license file 👍 
![404](https://github.com/2004-okari/TODOLIST/assets/89055502/5490d195-194f-49a4-b52c-277be64d1f21)